components/create_post/create_post.jsx

```diff
@@ -306,6 +308,7 @@ class CreatePost extends React.PureComponent {
             currentChannel: props.currentChannel,
         };
 
+        this.props.actions.setShowPreview(false);
```

Comment:
Similarly, this should be called in `componentDidMount` and not the constructor.